Earthfile

```diff
@@ -674,17 +674,28 @@ webui-deps:
 docs:
     FROM node:19-bullseye
     ARG USERARCH
+    ARG MODEL
+    IF [ "$MODEL" = "rpi64" ] || [ "$USERARCH" = "arm64" ] || [ "$USERARCH" = "linux/arm64" ]
```

Comment:
you should be able to use the auto-define `TARGETARCH` (arm, amd64, arm64) which is set from earthly directly to identify the arch:
https://docs.earthly.dev/docs/earthfile/builtin-args

Reply:
Oh I didn't see this one, thanks!